### ISSUE TITLE
mu doesn't contain the binaries msg2pdf or mug

### ISSUE THREAD

**Issue by simendsjo** [NixOS/nixpkgs#48005]
## Issue description

`mu` doesn't package msg2pdf or mug.

### Steps to reproduce

Install mu. Look in the bin folder, which should contain msg2pdf and mug (according to the nix expression)

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.74, NixOS, 18.09.783.299814b385d (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.1`
 - channels(simendsjo): `""`
 - channels(root): `"nixos-18.09.783.299814b385d"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by veprbl** [comment]
mu is built without mug by default. You can do an override: `mu.override { withMug = true; }` (it will ask you to add a security exception for `webkitgtk` also). When you do that, then `msg2pdf` and `mug` should appear in `bin/`.

**Comment by simendsjo** [comment]
Thanks. Trying it now. Seems it have to build webkit from scratch, so this will probably take a couple of hours.

**Comment by simendsjo** [comment]
Unfortunately, it doesn't build correctly:

```
make  all-recursive
make[4]: Entering directory '/build/source/toys/mug'
Making all in .
make[5]: Entering directory '/build/source/toys/mug'
  CC       mug.o
  CC       mug-msg-list-view.o
  CC       mug-msg-view.o
  CC       mug-query-bar.o
  CC       mug-shortcuts.o
  CXX      dummy.o
  CC       mu-widget-util.lo
  CC       mu-msg-attach-view.lo
  CC       mu-msg-body-view.lo
  CC       mu-msg-header-view.lo
  CC       mu-msg-view.lo
  CCLD     libmuwidgets.la
libtool: warning: '/nix/store/0i31k62b14rwfxx3fldm3ivl2iwvp0dc-harfbuzz-icu-1.8.8/lib/libharfbuzz.la' seems to be moved
ar: `u' modifier ignored since `D' is the default (see `U')
  CXXLD    mug
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstapp-1.0
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstaudio-1.0
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstfft-1.0
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstpbutils-1.0
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstvideo-1.0
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstbase-1.0
/nix/store/h0lbngpv6ln56hjj59i6l77vxq25flbz-binutils-2.30/bin/ld: cannot find -lgstreamer-1.0
collect2: error: ld returned 1 exit status
make[5]: *** [Makefile:552: mug] Error 1
make[5]: Leaving directory '/build/source/toys/mug'
make[4]: *** [Makefile:633: all-recursive] Error 1
make[4]: Leaving directory '/build/source/toys/mug'
make[3]: *** [Makefile:492: all] Error 2
make[3]: Leaving directory '/build/source/toys/mug'
make[2]: *** [Makefile:400: all-recursive] Error 1
make[2]: Leaving directory '/build/source/toys'
make[1]: *** [Makefile:519: all-recursive] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:429: all] Error 2
builder for '/nix/store/ygn4bp90xpxcqz2kn4k4jbklxbxbh15z-mu-1.0.drv' failed with exit code 2
cannot build derivation '/nix/store/2gnzikgv65raspgi77vkfqhg0pg9hhmd-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/w5s6gvkhfhi1sb7kp59cm67x9nng8fhg-nixos-system-simendsjo-macbook-18.09.812.8cfce968b1a.drv': 1 dependencies couldn't be built
error: build of '/nix/store/w5s6gvkhfhi1sb7kp59cm67x9nng8fhg-nixos-system-simendsjo-macbook-18.09.812.8cfce968b1a.drv' failed
```

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.74, NixOS, 18.09.812.8cfce968b1a (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.1`
 - channels(simendsjo): `""`
 - channels(root): `"nixos-18.09.812.8cfce968b1a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by veprbl** [comment]
I think it did build for me on linux. I can’t tell you which version of nixpkgs it was, right now.

**Comment by veprbl** [comment]
`mu` builds for me with `withMug = true;` on nixpkgs ea691fc8ee9cb4b49342148bac05b73e5068323f, I can also see mug and msg2pdf in bin.

**Comment by xeji** [comment]
fixed in #48830
